### PR TITLE
Add Service Offering to listSystemVMs and fix link from VR to its offering

### PR DESCRIPTION
**Pull request by gpordeus**
### Description

The API call `listSystemVMs` does not return the VM's service offering like `listVirtualMachines` does and, as such, the system VM page in the UI doesn't have that information like a user VM page has. This PR adds the fields `serviceOfferingId` and `serviceOfferingName` to the `SystemVmResponse` class.

It also changes the link to the offering in the VR page from `client/#/computeoffering/<uuid>?issystem=true` to `/client/#/systemoffering/<uuid>` for consistency (since it was the only place with such a way to access the system offering).

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

### How Has This Been Tested?
I accessed the SSVM and CPVM pages in the UI and saw the field "Compute offering" with the name and link to their respective offerings. If the offering does not exist, it returns 404 in the same way a missing template does.

On the VR page, the link to its offering now redirects to the same page as if you had gone through Service Offerings -> System offerings.

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]
clgtm, @gpordeus can you add and integration test for this? There is probably one you can extend to include a check for these fields.

File: api/src/main/java/org/apache/cloudstack/api/response/SystemVmResponse.java [hunk 1]
```diff
@@ -178,6 +178,14 @@ public class SystemVmResponse extends BaseResponseWithAnnotations {
     @Param(description = "true if vm contains XS/VMWare tools inorder to support dynamic scaling of VM cpu/memory.")
     private Boolean isDynamicallyScalable;
 
+    @SerializedName(ApiConstants.SERVICE_OFFERING_ID)
+    @Param(description = "the ID of the service offering of the system virtual machine")
```
Comment (JoaoJandre) [inline api/src/main/java/org/apache/cloudstack/api/response/SystemVmResponse.java:182]:
```suggestion
    @Param(description = "the ID of the service offering of the system virtual machine.")
```

File: api/src/main/java/org/apache/cloudstack/api/response/SystemVmResponse.java [hunk 2]
```diff
@@ -178,6 +178,14 @@ public class SystemVmResponse extends BaseResponseWithAnnotations {
...
+    private String serviceOfferingId;
+
+    @SerializedName("serviceofferingname")
+    @Param(description = "the name of the service offering of the system virtual machine")
```
Comment (JoaoJandre) [inline api/src/main/java/org/apache/cloudstack/api/response/SystemVmResponse.java:186]:
```suggestion
    @Param(description = "the name of the service offering of the system virtual machine.")
```

**Review by JoaoJandre — APPROVED** [review]
LGTM
Did some testing in the UI and the offerings where shown as expected for CPVM, SSVM and VRs.

**Pull request merged**